docs/import/manage_disaster.js

```diff
@@ -804,9 +829,25 @@ function createAssetDropdown(
     }
     select.append(assetOption);
   }
+  select.on(
+      'change', (event) => handleScoreAssetSelection(event, propertyPath));
+
   return select;
 }
 
+/**
+ * Handles the user selecting an asset for one of the possible score types.
+ * @param {Event} event Event, used to identify the target
+ * @param {Array<string>} propertyPath path to property inside asset data. We
+ *     set this value by setting the parent's attribute to the target's value
+ */
+function handleScoreAssetSelection(event, propertyPath) {
+  const parentProperty = getElementFromPath(propertyPath.slice(0, -1));
```

Comment:
Maybe add some logic commentary?

Also, wondering how intuitive it is for changing the value of the drop down to trigger a write. I guess changing the disaster drop down toggles something but that's visually clear where for other drop downs you have to click a button to trigger anything. Maybe a TODO for later to have some visual indication that this changed something.

Reply:
This is by analogy with the manage_layers page, where changing the value of drop-downs also triggers writes, no?

Added logic explanation.